common/static/coffee/src/discussion/views/discussion_thread_edit_view.js

```diff
@@ -53,13 +58,14 @@
                 var title = this.$('.edit-post-title').val(),
                     threadType = this.$(".post-type-input:checked").val(),
                     body = this.$('.edit-post-body textarea').val(),
-                    commentableId = this.topicView.getCurrentTopicId(),
                     postData = {
                         title: title,
                         thread_type: threadType,
-                        body: body,
-                        commentable_id: commentableId
+                        body: body
                     };
+                if (this.topicView) {
```

Comment:
Nit: could this conditional be `if ( this.context === 'course')`? As far as I can tell it's logically equivalent and makes it a bit more clear.